tests/gui/anaconda-autogui-testing.ks

```diff
@@ -10,7 +10,7 @@ rootpw qweqwe
 bootloader --location=mbr
 zerombr
 clearpart --all
-part / --fstype="ext4" --size=6000
+part / --fstype="ext4" --size=10240
```

Comment:
The same ks file is used to create the LiveCD and the available space was not enough. 10G is probably too much though.